getting_started_moa/creating-first-moa-cluster.adoc

```diff
@@ -0,0 +1,46 @@
+:context: creating-first-moa-cluster
+[id="creating-first-moa-cluster"]
+= Getting started with {product-title}
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+This guide walks through setting up your first {product-title} cluster using `moactl`.
+
```

Comment:
I like this TL;DR version for a user that doesn't want to walk through the full tutorial and feels confident using the below commands.